src/api/schema/typePolicies/typePolicies.base.ts

```diff
@@ -79,4 +79,27 @@ export const typePolicies: TypePolicies = {
       },
     },
   },
+  Partner: {
+    fields: {
+      projects: {
+        merge(existing, incoming: SecuredProjectList) {
```

Comment:
Adding this custom merge function to the type policies along with "    fetchPolicy: 'cache-first'," to the query is producing the desired behavior for server sort / pagination. That is:

1. New network calls are made for a sort / pagination combo that has not yet been fetched
2. Calls for previously fetched sort / pagination combos are taken from apollo cache with no network call
3.  The apollo caching warning I mentioned is gone

That said I would have preferred not to have to do this custom logic to achieve this result. The [apollo docs](https://www.apollographql.com/docs/react/v2/data/pagination/) on this matter say that the natural behavior of thier caching mechanism is as follows:

"Once the new data is returned from the server, the updateQuery function is used to merge it with the existing data, which will cause a re-render of your UI component with an expanded list."

That is why I was seeing items add to the re-render instead of overwriting the previous page. I see no documentation around how to do what we are trying to do (some tutorials are there but only for client pagination). I even tossed a generic version of the issue to [stackoverfow 
](https://stackoverflow.com/questions/77813536/apollo-server-pagination-list-only-items-recently-fetched-from-cache-or-network/77822005#77822005) and got crickets lol (I am going to post a bounty later)

@CarsonF If you have a better way I'd love to implement it... Otherwise this wors without error and doesn't seem to create a serious readability issue or any performance issues at all